### PR TITLE
refa(docs): update mdx pages with name a-c

### PR DESCRIPTION
**Pull request by ti10le**
#1888

### CODE REVIEW

File: docs/content/components/action-group.mdx [hunk 1]
```diff
@@ -6,56 +6,44 @@ import { FigmaLink } from '../../src/components/FigmaLink';
...
+```tsx onlyCode
+import { ActionGroup } from '@marigold/components';
+```
+
 <FigmaLink
   to={
     'https://www.figma.com/file/DFKyTGHAoDxOsUBPszLLxP/%F0%9F%8F%B5%EF%B8%8FMarigold?node-id=864%3A1'
   }
 />
+<br />
```
Comment (sebald) [inline docs/content/components/action-group.mdx:33]: ?

File: docs/content/components/action-group.mdx [hunk 2]
```diff
@@ -6,56 +6,44 @@ import { FigmaLink } from '../../src/components/FigmaLink';
...
+  <ActionGroup space="xsmall">
+    <Button variant="text" size="small">
+      Cancel
+    </Button>
+    <Button variant="primary" size="small">
+      Confirm
+    </Button>
+  </ActionGroup>
+</MarigoldTheme>
+<br />
```
Comment (sebald) [inline docs/content/components/action-group.mdx:21]: ?

File: docs/src/components/FigmaLink.tsx [hunk 3]
```diff
@@ -1,39 +1,44 @@
...
-        fill="#FF5252"
-      />
-      <path
-        d="M9.44253 7.08183C9.44253 8.3849 8.38497 9.44247 7.08189 9.44247C5.77882 9.44247 4.72125 8.3849 4.72125 7.08183C4.72125 5.77876 5.77882 4.72119 7.08189 4.72119C8.38497 4.72119 9.44253 5.77876 9.44253 7.08183Z"
-        fill="#00BBF5"
-      />
-    </svg>
-    View in Figma
-  </Link>
+  <Box py="small">
```
Comment (sebald) [inline docs/src/components/FigmaLink.tsx:11]: Why? 🤔

**Review by sebald — COMMENTED** [review]
Can we split this in more digestible pieces 🙈

File: docs/src/components/FigmaLink.tsx (hunk 3 above)
Comment (ti10le) [reply docs/src/components/FigmaLink.tsx:11]: looks not good without space between. How would you handle this?

File: docs/content/components/action-group.mdx [hunk 4]
```diff
@@ -6,56 +6,39 @@ import { FigmaLink } from '../../src/components/FigmaLink';
...
-    Cancel
-  </Button>
-  <Button variant="secondary" size="small">
-    Delete <Delete />
-  </Button>
-</ActionGroup>
-```
-
-### With space
+Space between the elements
```
Comment (sebald) [inline docs/content/components/action-group.mdx:41]: Heading is missing

File: docs/content/components/action-group.mdx [hunk 5]
```diff
@@ -64,7 +47,7 @@ import { ActionGroup } from '@marigold/components';
 </ActionGroup>
 ```
 
-### With vertical alignment
+Vertical alignment
```
Comment (sebald) [inline docs/content/components/action-group.mdx:50]: Heading is missing

File: docs/content/components/action-group.mdx [hunk 6]
```diff
@@ -76,3 +59,16 @@ import { ActionGroup } from '@marigold/components';
   </Button>
 </ActionGroup>
 ```
+
+## Group two Buttons in a form
```
Comment (sebald) [inline docs/content/components/action-group.mdx:63]: 3 #!

File: docs/content/components/aside.mdx [hunk 7]
```diff
@@ -29,29 +29,22 @@ It is important to note that the `<Aside>` must have exactly too children, where
...
+## Examples
 
-```tsx onlyCode
-import { Aside } from '@marigold/components';
-```
-
-## Types
-
-### Spacing
+Space between elements
```
Comment (sebald) [inline docs/content/components/aside.mdx:34]: Heading is missing

File: docs/content/components/aside.mdx [hunk 8]
```diff
@@ -29,29 +29,22 @@ It is important to note that the `<Aside>` must have exactly too children, where
...
-    <Box border="1px solid #ced4da" bg="#e9ecef" />
-  </Aside>
+<Aside space="medium">
+  <Box border="1px solid #ced4da" bg="#e9ecef" height="100px" width="50px" />
+  <Box border="1px solid #ced4da" bg="#e9ecef" />
+</Aside>
 ```
 
-### Aside on the right
+Aside on the right
```
Comment (sebald) [inline docs/content/components/aside.mdx:43]: Heading is missing

File: docs/content/components/aspect.mdx [hunk 9]
```diff
@@ -45,7 +39,7 @@ import { Aspect } from '@marigold/components';
 </Aspect>
 ```
 
-### Map Ratio Landscape
+## Map
```
Comment (sebald) [inline docs/content/components/aspect.mdx:42]: 3?

File: docs/content/components/aspect.mdx [hunk 10]
```diff
@@ -58,7 +52,7 @@ import { Aspect } from '@marigold/components';
 </Aspect>
 ```
 
-### Video - Ratio Widescreen
+## Video
```
Comment (sebald) [inline docs/content/components/aspect.mdx:55]:
```suggestion
### Video
```

**Review by sebald — CHANGES_REQUESTED** [review]
I am not sure but there are missing a lot of headings?

**Comment by ti10le** [comment]
ok @sebald then I would do Examples allways with 2# and all after with 3#.
Looks like the template isn't good enough 👀

**Comment by sebald** [comment]
@ti10le what would you improve? The example use case in there uses ###

**Comment by ti10le** [comment]
> @ti10le what would you improve? The example use case in there uses ###

Yes a little bit more clear when to use which heding. I will do it 😊  Then we have it the same like all examples after merching

**Pull request merged**